playbooks/ci-ceph_swift/tasks/pre-deploy.yml

```diff
@@ -35,6 +35,32 @@
     - name: umount ephemeral disk on ceph_osds
       shell: umount /dev/{{ item }}
       with_items: "{{ ceph.disks }}"
+      ignore_errors: true
+
+- name: umount ephemeral disk on swiftnode
+  hosts: swiftnode
+  tasks:
+    - name: umount ephemeral disk on swiftnode
+      shell: umount /dev/{{ item.disk }}
+      with_items: "{{ swift.disks }}"
+      ignore_errors: true
+
+- name: create new partition table for ephemeral disk on swiftnode
+  hosts: swiftnode
+  tasks:
+    - name: create new msdos partition table for ephemeral disk
+      shell: parted /dev/{{ item.disk }} --script mklabel msdos
+      with_items: "{{ swift.disks }}"
+
+- name: deploy fake lshw file on swiftnode
+  hosts: swiftnode
+  tasks:
+    - name: rename the original /usr/bin/lshw
+      command: mv /usr/bin/lshw /usr/bin/lshw.original
+    - name: deploy fake lshw file /usr/bin/lshw
+      template: src=../templates/usr/bin/lshw
+                dest=/usr/bin/lshw
+                owner=root group=root mode=0755
```

Comment:
After a testing with update-alternatives, I found that it is not suitable here, detailed reason is:
```
ubuntu@ceph-swift-test-swift-0:~$ sudo cat /etc/sudoers.d/swiftops
swiftops ALL= NOPASSWD: /usr/local/bin/swifttool
swiftops ALL= NOPASSWD: /usr/bin/lshw
swiftops ALL= NOPASSWD: /usr/local/bin/swift-init
```
Our alternatives for lshw /usr/local/bin/lshw is not registered in above file which defines the sudoer permission of swiftops user, so password was required during running swifttool with user swiftops.
The solution is to use my previous approach to rename original /usr/bin/lshw to /usr/bin/lshw.original and wrap it in my fake one /usr/bin/lshw.